hocon.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name        = 'hocon'
-  s.version     = '1.1.2'
+  s.version     = '1.1.2-SNAPSHOT'
```

Comment:
Oh I just ending up adding this so I could differentiate the version number when I install the gem. I'll probably just take it out since like you said, I don't think SNAPSHOT means anything to ruby. I'll bump the version